components/staking/ValidatorRow.vue

```diff
@@ -147,7 +147,7 @@ h5 {
 .validator-image {
   border-radius: 50%;
   height: 2.5rem;
-  width: 2.5rem;
+  width: 2.5rem !important;
```

Comment:
@jbibla

Reply:
not necessary, please remove @Bitcoinera - this will be addressed in another PR

Reply:
but it looks very ugly. Too big. Why not fix it here?

Reply:
"!important" is a value that you try to avoid at all costs as it can conflict with other styles. identifying the need for "!important" leads towards a problem up the structure.

Reply:
Ok, I see. Thank you